proto/test_proto/test.proto

```diff
@@ -361,67 +361,9 @@ extend DefaultsMessage {
   optional DefaultsMessage.DefaultsEnum default_enum = 216 [default = ONE];
 }
 
-message MyMessageSet {
-  option message_set_wire_format = true;
-  extensions 100 to max;
-}
```

Comment:
It was necessary to delete this plus all references to it because message sets are a protov1 feature unsupported by the official protobuf and so registering this file descriptor was failing